src/ResourceManager/IotHub/Commands.IotHub.Test/ScenarioTests/IotHubTests.ps1

```diff
@@ -37,8 +37,50 @@ function Test-AzureRmIotHubLifecycle
 	# Create or Update Resource Group
 	$resourceGroup = New-AzureRmResourceGroup -Name $ResourceGroupName -Location $Location 
 
+	Write-Debug " Create new eventHub " 
+	$namespaceName = "IotHubPSEHNamespaceTest"
```

Comment:
I am suggesting generating the parameter based on a prefix, there are lots of examples of using the test library to do this.